src/frontend/next/src/styles/telescope-post-content.css

```diff
@@ -19,7 +19,7 @@
   max-width: 100%;
   padding-top: 1.5rem;
   padding-bottom: 1rem;
-  width: 100%;
+  display: flex;
 }
```

Comment:
with ```max-width: 100%```
Then you just need ```height: auto```

Reply:
Thank you for the comment! But what exactly does `height: auto` do in this case? I can't see it affect the image size (correct me if I'm wrong)